packages/components/src/internal/components/samples/constants.ts

```diff
@@ -211,3 +211,61 @@ export const SAMPLE_DATA_EXPORT_CONFIG = {
 // Issue 46037: Some plate-based assays (e.g., NAB) create samples with a bogus 'Material' sample type, which should get excluded everywhere in the application
 export const SAMPLES_WITH_TYPES_FILTER = Filter.create('SampleSet', 'Material', Filter.Types.NEQ);
 export const NON_MEDIA_SAMPLE_TYPES_FILTER = Filter.create('Category', 'media', Filter.Types.NEQ_OR_NULL);
+
+export const SAMPLE_DOMAIN_DEFAULT_SYSTEM_FIELDS = [
+    {
+        Name: 'Name',
+        Label: 'Sample ID',
+        'Data Type': 'Text',
+        Required: false,
```

Comment:
similar comment here about the Name being required: true and about the descriptions ending in periods.